src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -690,30 +734,30 @@ def test_linux_webapp(self, resource_group):
             resource_group, webapp)).get_output_in_json()
         self.assertEqual(result2, [])
 
-
-class LinuxWebappSSHScenarioTest(ScenarioTest):
-    @ResourceGroupPreparer(location='japanwest')
-    def test_linux_webapp_ssh(self, resource_group):
-        # On Windows, test 'webapp ssh' throws error
-        import platform
-        if platform.system() == "Windows":
-            from azure.cli.core.util import CLIError
-            with self.assertRaises(CLIError):
-                self.cmd('webapp ssh -g {} -n {} --timeout 5'.format("foo", "bar"))
-            return
-
-        runtime = 'node|8.11'
-        plan = self.create_random_name(prefix='webapp-ssh-plan', length=24)
-        webapp = self.create_random_name(prefix='webapp-ssh', length=24)
-        self.cmd(
-            'appservice plan create -g {} -n {} --sku S1 --is-linux' .format(resource_group, plan))
-        self.cmd('webapp create -g {} -n {} --plan {} --runtime {}'.format(
-            resource_group, webapp, plan, runtime))
-        time.sleep(30)
-        requests.get('http://{}.azurewebsites.net'.format(webapp), timeout=240)
-        time.sleep(30)
-        self.cmd('webapp ssh -g {} -n {} --timeout 5'.format(resource_group, webapp))
-        time.sleep(30)
+# This is failing on windows OS. Rised a bug #12844 to fix in future releases
+# class LinuxWebappSSHScenarioTest(ScenarioTest):
```

Comment:
You can add a `@unittest.skip()`

Reply:
> You can add a @unittest.skip() instead of adding so many #

did the change as said above. please review.